chain/src/chain.rs

```diff
@@ -246,54 +246,52 @@ impl Chain {
 
 				Ok(head)
 			}
-			Err(e) => {
```

Comment:
Ignore changes in this file. This is all just rustfmt deciding to do it now, not sure why...